msteams-platform/m365-apps/prerequisites.md

```diff
@@ -87,10 +87,16 @@ To install Microsoft 365 applications in your test environment:
     |Outlook Add-ins | `BetaChannel` |
     |Meeting apps in Outlook | `Current` |
 
+   | **Word, Excel, PowerPoint scenario** | **Update channel** |
+    |---------------|--------------|
+   |Word, Excel, PowerPoint Add-ins | `BetaChannel` |
+   
+   
+
 1. Open Command Prompt and go to the local folder path.
 1. Run `setup.exe /configure configuration-Office365-x86.xml` (or use the **x64.xml* file, depending on your setup).
-1. Open Outlook (desktop client) and set up the mail account using your test tenant credentials.
-1. Open **File** > **Office Account** > **About Outlook** to confirm you're running a Microsoft 365 *Current Channel* build of Outlook.
+1. Open Outlook, Word, Excel and PowerPoint (desktop client) and set up the mail account using your test tenant credentials.
+1. If you're testing in Outlook, open **File** > **Office Account** > **About Outlook** to confirm you're running a Microsoft 365 *Current Channel* build. If you're testing in Word, Excel, or PowerPoint, open **File** > **Account** > **Update Channel** to confirm you're running a Microsoft 365 *Beta Channel* build.
```

Comment:
```suggestion
1. If you are testing in Outlook, take the following steps:
    1. Open the Outlook desktop client and set up the mail account using your test tenant credentials.
    1. In Outlook, open **File** > **Office Account** > **About Outlook** to confirm you're running a Microsoft 365 *Current Channel* build. 
1. If you're testing in Word, Excel, or PowerPoint, open **File** > **Account** > **Update Channel** to confirm you're running a Microsoft 365 *Beta Channel* build.
```

Reply:
> 

I cannot approve unless this change is made. The current step 7 says "Open Outlook, Word, Excel and PowerPoint (desktop client) and set up the mail account using your test tenant credentials." There's no reason to open all 4 of them. Also, it makes no sense to open Word, Excel, or PowerPoint and then do something with a **mail** account.

Reply:
@LuyiH had approved this change (via Teams conversation), @V-bvishnu5 are you able to accept the change so we can unblock @Rick-Kirkham's approval? (I don't have permissions). Thanks!

Reply:
@v-shalinir  This change has been marked "resolved" twice now without anyone actually making the change. For the reasons I gave, the change has to be made.

Reply:
@Rick-Kirkham, apologies, the comment was marked resolved by mistake. Requesting @LuyiH to please update the comment asap. We will only merge this PR after @Rick-Kirkham's approval.